service/src/lib.rs

```diff
@@ -544,6 +546,22 @@ impl Service for sync_wrapper::SyncWrapper<axum::Router> {
     }
 }
 
+#[cfg(feature = "web-actix-web")]
+#[async_trait]
+impl<F> Service for F
+where F: FnOnce(&mut actix_web::web::ServiceConfig) + Sync + Send + Copy + Clone + 'static {
+    async fn bind(self: Box<Self>, addr: SocketAddr) -> Result<(), Error> {
+        let srv = actix_web::HttpServer::new(move || actix_web::App::new().configure(*self))
+            .bind(addr)?
+            .run();
```

Comment:
Oh, nice!! Quite a few of us tried an implementation before but could not get it working.

Reply:
@chesedo Yep, `cargo shuttle run` works! I've implemented the example, but haven't created a PR yet.